src/ItemsExplorer/SitemapPartialProvider.ts

```diff
@@ -42,7 +45,7 @@ export class SitemapPartialProvider {
         if (document.fileName.endsWith('sitemap')) {
             editor.insertSnippet(SNIPPET_TEMPLATE(this.item), editor.selection.active)
         } else {
-            window.showInformationMessage('Please open "*.sitemap" file in the editor to add a new snippet.')
+            window.showInformationMessage(localize('addToSitemap.wrongFileEnding.text','Please open "*.sitemap" file in the editor to add a new snippet.'))

```

Comment:
Please add a space after `,`. Ideally reformat the whole file (Alt + Shift + F) if you have Prettier installed it should work right away.